tests/01_03_52-disable_lock_whitelist.js

```diff
@@ -1,121 +1,107 @@
+const { getBtcClient } = require('../lib/btc-client-provider');
+const { getBridge, getLatestActiveForkName } = require('../lib/precompiled-abi-forks-util');
+const { satoshisToBtc, btcToSatoshis, satoshisToWeis } = require('@rsksmart/btc-eth-unit-converter');
+const { sendPegin, ensurePeginIsRegistered } = require('../lib/2wp-utils');
+const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
 
-var federationAddress;
-var btcClient;
-var rskClient;
-var rskClients;
-var pegClient;
-var test;
-var utils;
-let rskTxHelpers;
-
-const NETWORK = bitcoin.networks.testnet;
-
-const INITIAL_BTC_BALANCE = bitcoin.btcToSatoshis(10);
+const { WHITELIST_CHANGE_PK, WHITELIST_CHANGE_ADDR} = require('../lib/assertions/whitelisting')
 
-const WHITELIST_CHANGE_PK = '3890187a3071327cee08467ba1b44ed4c13adb2da0d5ffcc0563c371fa88259c';
-const WHITELIST_CHANGE_ADDR = '87d2a0f33744929da08b65fd62b627ea52b25f8e';
+let rskTxHelpers;
+let rskTxHelper;
+let btcTxHelper;
+let bridge;
+let federationAddress;
+let MINIMUM_PEGIN_VALUE_IN_BTC;
+let MINIMUM_PEGIN_VALUE_IN_SATS;
+let WHITELIST_DISABLE_BLOCK_DELAY;
+let FEE_IN_SATOSHI;
+
+const fulfillRequirementsToRunAsSingleTestFile = async () => {
+  await rskUtils.activateFork(Runners.common.forks.papyrus200);
+};
 
 describe('Disable whitelisting', function() {
-  var addresses;
 
   before(async () => {
-    try{
-      btcClient = bitcoin.getClient(
-        Runners.hosts.bitcoin.rpcHost,
-        Runners.hosts.bitcoin.rpcUser,
-        Runners.hosts.bitcoin.rpcPassword,
-        NETWORK
-      );
-      rskClient = rsk.getClient(Runners.hosts.federate.host);
-      rskClients = Runners.hosts.federates.map(federate => rsk.getClient(federate.host));
-      pegClient = pegUtils.using(btcClient, rskClient);
-      test = pegAssertions.with(btcClient, rskClient, pegClient, rskClients);
-      utils = rskUtilsLegacy.with(btcClient, rskClient, pegClient);
-      rskTxHelpers = getRskTransactionHelpers();
-
-      // Grab the federation address
-      federationAddress = await rskClient.rsk.bridge.methods.getFederationAddress().call();
-      await btcClient.importAddress(federationAddress, 'federations');
-      
-      addresses = await pegClient.generateNewAddress('test');
-      expect(addresses.inRSK).to.be.true;
-      
-      await btcClient.sendToAddress(addresses.btc, INITIAL_BTC_BALANCE);
-      await btcClient.generate(1);
-      await test.assertBitcoinBalance(addresses.btc, INITIAL_BTC_BALANCE, 'Initial BTC balance');
-      
-      var addr = await rskClient.eth.personal.importRawKey(WHITELIST_CHANGE_PK, '');
-      expect(addr.slice(2)).to.equal(WHITELIST_CHANGE_ADDR);
-      
-      await rskClient.eth.personal.unlockAccount(addr, '');
-      await sequentialPromise(10, () => rskUtils.mineAndSync(rskTxHelpers));
-    }
-    catch (err) {
-      throw new CustomError('Lock whitelisting failure', err);
+    if(process.env.RUNNING_SINGLE_TEST_FILE) {
```

Comment:
Put this block of code above the `getBridge(...`) part. It may be working now for some reason, but we need to activate the fork first before getting the bridge with the latest fork, so we create the bridge with the fork that we need and not some other fork.

Reply:
it's like that already